### PR TITLE
[IMP] l10n_lv: standardize basic package

### PR DESCRIPTION
**Pull request by mairasalazar**
Documentation found on the Latvian chart of accounts allowed to improve language used in the Chart of Accounts, as well as remove/add some accounts.

Additionally:

- Changes were made to the taxes, to differentiate between goods and services.
- Renamed tax report file.
- Added report menu.
- Update translation file.

task-3640443

### CODE REVIEW

File: addons/l10n_lv/i18n/l10n_lv.pot [hunk 1]
```diff
@@ -0,0 +1,243 @@
+# Translation of Odoo Server.
```
Comment (malb-odoo) [inline addons/l10n_lv/i18n/l10n_lv.pot:1]: For localisation especially, a pot without it's po file is useless. So you will need to do the po in latvia :smile: To do this you can do it from the interface by installing the lang you want to export then going in the settings -> translation -> export translation -> select the language and the app you want to export and hop you can download the po file

**Review by malb-odoo — CHANGES_REQUESTED** [review]
LGTM thanks for the work on the localisation ! Just the missing po file and it will be good to go for me :smile:

File: addons/l10n_lv/data/template/account.fiscal.position-lv.csv [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
-"fiscal_position_ex","Outside EU","1","","","40","","tax_sale_vat_5","tax_sale_vat_0_ex_g"
-"","","","","","","","tax_sale_vat_12","tax_sale_vat_0_ex_g"
-"","","","","","","","tax_sale_vat_reverse","tax_sale_vat_0_ex_g"
-"","","","","","","","tax_sale_vat_21","tax_sale_vat_0_ex_g"
-"","","","","","","","tax_purchase_vat_21","tax_purchase_vat_21_ex_g"
-"","","","","","","","tax_purchase_vat_12","tax_purchase_vat_21_ex_g"
-"","","","","","","","tax_purchase_vat_5","tax_purchase_vat_21_ex_g"
+"id","name","auto_apply","vat_required","country_id","sequence","country_group_id","tax_ids/tax_src_id","tax_ids/tax_dest_id","name@lv"
+"fiscal_position_lv_domestic","Domestic","1","1","base.lv","10","","","","Iekšzemes"
+"fiscal_position_lv_eu","EU Companies","1","1","","20","base.europe","VAT_S_G_21_LV","VAT_S_EU_G_0_LV","ES uzņēmumi"
```
Comment (malb-odoo) [inline addons/l10n_lv/data/template/account.fiscal.position-lv.csv:3]: Translation for EU is a bit weird ? i would keep EU and not ES :smile:

File: addons/l10n_lv/data/template/account.fiscal.position-lv.csv [hunk 3]
```diff
@@ -1,17 +1,29 @@
...
+"","","","","","","","VAT_S_S_12_LV","VAT_S_EU_S_0_LV",""
+"","","","","","","","VAT_S_G_5_LV","VAT_S_EU_G_0_LV",""
+"","","","","","","","VAT_S_S_5_LV","VAT_S_EU_S_0_LV",""
+"","","","","","","","VAT_P_G_21_LV","VAT_P_EU_G_21_LV",""
+"","","","","","","","VAT_P_S_21_LV","VAT_P_EU_S_21_LV",""
+"","","","","","","","VAT_P_G_12_LV","VAT_P_EU_G_12_LV",""
+"","","","","","","","VAT_P_S_12_LV","VAT_P_EU_S_12_LV",""
+"","","","","","","","VAT_P_G_5_LV","VAT_P_EU_G_5_LV",""
+"","","","","","","","VAT_P_S_5_LV","VAT_P_EU_S_5_LV",""
+"fiscal_position_lv_eu_private","EU Consumer","1","","","30","base.europe","","","ES patērētājs"
```
Comment (malb-odoo) [inline addons/l10n_lv/data/template/account.fiscal.position-lv.csv:16]: same for here and the others :smile:

**Review by malb-odoo — APPROVED** [review]
Thanks for the changes ! Very nice job on this :smile:

**Comment by qdp-odoo** [comment]
@robodoo r+

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**